Add section to Frequently Asked Questions - Profiles Sync (#4657)
segmentio/segment-docs: src/unify/profiles-sync/sample-queries.md

- If you rebuild or use non-incremental materialization for `profile_traits`, Profiles Sync will fully calculate traits against a user. As a result, Profiles Sync would ensure that all traits reflect the most recently observed value for fully-merged users.

By contrast, Segment Unify and incrementally-built Profiles Sync materializations won’t combine already-computed traits across two merged profiles at the moment of merge. Instead, one profile’s traits will be chosen across the board.

> #### What hash function is used for the external_id_hash field by Profile Sync?

Reviewer: ```suggestion
#### What hash function is used for the external_id_hash field by Profiles Sync?
```

> The `external_id_hash` is a hash of the `external_id_type` and `external_id_value` using SHA-1. This field corresponds to the `primary_key` for the table: `hash (external_id_type and external_id_value)`

Reviewer: ```suggestion
The `external_id_hash` is a hash of the `external_id_type` and `external_id_value` using SHA-1. This field corresponds to the `primary_key` for the table: `hash (external_id_type and external_id_value)`.
```

> The value of the `external_id_hash` is a hash of the `external_id_type` and `external_id_value`, using SHA-1. This field corresponds the `primary_key` for the table. `hash ( external_id_type and external_id_value)`

Reviewer: ```suggestion
The `external_id_hash` is a hash of the `external_id_type` and `external_id_value` using SHA-1. This field corresponds to the `primary_key` for the table: `hash (external_id_type and external_id_value)`
```

> For example, in Big Query this is the logic, `TO_HEX(SHA1(concat(external_id_type, external_id_value))) as seg_hash`.

Reviewer: ```suggestion
For example, in BigQuery the logic is: `TO_HEX(SHA1(concat(external_id_type, external_id_value))) as seg_hash`.
```
